shynet/dashboard/templatetags/helpers.py

```diff
@@ -184,3 +184,9 @@ def urldisplay(url):
         )
     else:
         return url
+
+
+@register.filter
+def percent(value, total):
+    percent = value / total
```

Comment:
Could we just add a quick check if `total` is `0`? So:

```suggestion
    if total == 0:
        return "N/A"
    percent = value / total
```